pyproject.toml

```
@@ -27,6 +27,7 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Physics",
 ]
 dependencies = [
+  'swxsoc @ git+https://github.com/swxsoc/swxsoc',
```

```suggestion
  'swxsoc @ git+https://github.com/swxsoc/swxsoc@main',
```
Quick change just to ensure it has the same dependency as when referenced in the pipeline